app/controllers/provider_interface/offer/conditions_controller.rb

```diff
@@ -7,7 +7,7 @@ def new
       end
 
       def create
-        @wizard = OfferWizard.new(offer_store, conditions_attributes_for_wizard)
+        @wizard = OfferWizard.new(offer_store, conditions_attributes_for_wizard.merge!(current_step: 'conditions'))
```

Comment:
Is it the case that the current step will be the same for all actions in each of the controllers? Could we move the `merge` to somewhere common?